L1Trigger/L1TTrackMatch/interface/METTrackTransform.h

```diff
@@ -0,0 +1,87 @@
+#ifndef L1Trigger_L1TTrackMatch_METTrackTransform_HH
+#define L1Trigger_L1TTrackMatch_METTrackTransform_HH
+
+#include "L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuAlgo.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+using namespace L1TkEtMissEmuAlgo;
+
+struct InternalEtWord{
```

Comment:
Why is this `InternalEtWord` defined both in this file and in L1TkEtMisEmuTrackTransorm? If this is really the same struct, then you should define it only in one location and then include it. Also, if this is internal to some other class/struct/namespace, then you should define it within that namespace.

Reply:
Fixed in latest commit, only L1TkEtMissEmuTrackTransform remains